src/services/review/RepoCheckerService.ts

```diff
@@ -819,6 +861,29 @@ export default class RepoCheckerService {
     })
   }
 
+  permalinkNormaliser(permalink: string) {
```

Comment:
nit: could we call this `normalisePermalink`

Reply:
[6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85)